src/minErr.js

```diff
@@ -63,3 +62,38 @@ function minErr(module, ErrorConstructor) {
     return new ErrorConstructor(message);
   };
 }
+
+/**
+ * @description
+ *
+ * In certain case (e.g. when catching and rethrowing an error), it is neither desirable nor
+ * necessary to pass the error through `minErr()`. You can use this function to avoid warnings
+ * produced by `ng-closire-runner` during `grunt minall`.
+ *
+ * Due to what arguments `ng-closure-runner` expects, the first two arguments must be static
+ * strings. Therefore, you have to pass the actual error as 3rd argument (see example below).
+ *
+ * **WARNING**
+ * Only use this function when you are certain that the thrown error should NOT be a `minErr`
+ * instance;
+ *
+ * Example usage:
+ *
+ * ```js
+ * try {
+ *   tryAndFail();
+ * } catch (err) {
+ *   doSomeThing(err);
+ *   throw noMinErr('', '', err);   // Functionally equivalent to `throw err`,
+ *                                  // but avoids `ng-closuer-runner` warnings.
+ * }
+ * ```
+ *
+ * @param {string} ignoredCode - Ignored, but necessary for `ng-closure-runner`.
+ * @param {string} ignoredTemplate - Ignored, but necessary for `ng-closure-runner`.
+ * @param {*} error - The error object that will be returned.
+ * @returns {*} - The passed in error.
+ */
+function noMinErr(ignoredCode, ignoredTemplate, err) {
```

Comment:
I am not thrilled about having to pass the first two string arguments.
Alternatively, we could teach `ng-closure-runner` to ignore `noMinErr` instances (i.e. not treat them as `minErr` instances, but don't log a warning either).